arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/TxInitializationSection.java

```diff
@@ -202,6 +202,12 @@ public void resolveAtEndTransaction(
       recipientUndoingValueReceptionNew =
           recipientValueReception.deepCopy().setDeploymentNumber(hub).turnOnWarmth();
 
+      if (tx.getTo().isEmpty()) {
+        recipientUndoingValueReception
+            .deploymentStatus(true)
+            .code(recipientValueReceptionNew.code());
+      }
+
```

Comment:
This is required deployment transactions get converted to either
- deployed byte code
- no deployment

and this requires a transition from the state of the account where there is initialization code to one where there isn't.